src/application-navigation/private/workspace-layout/WorkspaceLayout.module.scss

```diff
@@ -0,0 +1,102 @@
+// Themes
+@import '../clinical-lowlight-theme/WorkspaceLayout.module';
+@import '../orion-fusion-theme/WorkspaceLayout.module';
+
+:local {
+  .layout-container {
+    align-items: flex-start;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    height: 100%;
+    justify-content: stretch;
+    overflow: hidden;
+    position: relative;
+  }
+
+  .layout-body {
+    align-items: flex-start;
+    display: flex;
+    flex: 1 1 auto;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    height: 100%;
+    justify-content: stretch;
+    overflow: hidden;
+  }
+
+  .content-body {
+    flex: 1 1 320px;
+    height: 100%;
+    overflow: hidden;
+
+    .workspace-visible.workspace-split & {
+      flex: 1 1 auto;
+    }
+  }
+
+  .workspace-body {
+    display: none;
+    flex: 1 1 320px;
+    height: 100%;
+    overflow: hidden;
+    position: relative;
+    width: 100%;
+    z-index: 2;
+
+    .workspace-visible.workspace-split & {
+      flex: 0 0 auto;
+      width: 320px; // fix small width to 320
+    }
+
+    &.visible {
+      display: block;
+    }
+
+    &.overlay {
+      bottom: 0;
+      position: absolute;
+      right: 0;
+      top: 0;
+      width: calc(100% - 40px);
+    }
+  }
+
+  .workspace-inner {
+    height: 100%;
+    overflow: hidden;
+    position: relative;
+    width: 100%;
+    z-index: -1;
+  }
+
+  .workspace-overlay {
+    background-color: var(--terra-application-workspace-layout-workspace-overlay-background-color, rgba(0, 0, 0, 0.3));
+    bottom: 0;
+    display: block;
+    left: 0;
+    position: absolute;
+    right: 0;
+    top: 0;
+    z-index: 1;
+  }
+
+  .resize-overlay {
+    bottom: 0;
+    cursor: col-resize;
+    display: none;
+    left: 0;
+    position: absolute;
+    right: 0;
+    top: 0;
+    z-index: 5; // TODO 5 is very important for safari, validate final value for this
```

Comment:
Are we keeping 5 as the z-index here?

Reply:
I validated 5 does work. Anything below causes element highlighting when dragging the resize bar. I'll leave it at 5 and we can re-evaluate in the future if a better implementation presents itself.